bundle/compliance/cis_gcp/rules/cis_2_1/rule.rego

```diff
@@ -0,0 +1,35 @@
+package compliance.cis_gcp.rules.cis_2_1
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.if
+import future.keywords.in
+
+default cloud_logging_is_configured = false
+
+finding = result if {
+	data_adapter.is_policies_resource
+
+	result := common.generate_result_without_expected(
+		common.calculate_result(cloud_logging_is_configured),
+		input.resource,
+	)
+}
+
+cloud_logging_is_configured if {
+	policy := input.resource[_].iam_policy
+	has_read_write_logs(policy)
+	not has_exempted_members(policy)
+} else = false
+
+has_read_write_logs(policy) if {
+	log_types := {t | t = policy.audit_configs[i].audit_log_configs[j].log_type}
+	2 in log_types # "DATA_WRITE"
+	3 in log_types # "DATA_READ" 
```

Comment:
what about `ADMIN_READ`? 

> Ensure that **Admin Read**, Data Write, and Data Read are enabled for all Google Cloud services and that no exemptions are allowed.

Reply:
thanks. will add. missed it. a bit odd it's part of the manual audit but not the automated one.